sample_data_config/twitter-ads-api/twitter-ads-api.config.yaml

```diff
@@ -1,7 +1,25 @@
 twitterAdsApi:
-  - dataPipelineId: twitter_ads_api_campaign_elife_journal
+  # - dataPipelineId: twitter_ads_api_campaign_elife_journal
```

Comment:
should we enable this part to test that this is still working?

Reply:
It is working I have test it on my local and it is on staging too and working.